riseagain47/EPIC-Mail
# Feat(user) - send email to group - #164725731

#### What does this PR do?
- This PR will enable the user to send email to a group

#### Description of Task to be completed?
Have the following:
- group model
- group controller 
- group route
- group unit and integration tests

#### How should this be manually tested?
1. User should start the server with `npm run dev-start`
2. Open Postman and makes a `POST` HTTP request to `localhost:5000/api/v2/groups/:id/users
3. User should receive a success message upon success and send email to a group or get a no group response.

#### Any background context you want to provide?
User should that the server is running before making the `POST` HTTP request

#### What are the relevant pivotal tracker stories?
- #164725731 - Feature - User can send email to a group
